kb/2025-03-25/cve-2025-1974.md

```diff
@@ -33,9 +28,20 @@ kubectl -n kube-system get po -l"app.kubernetes.io/name=rke2-ingress-nginx" -ojs
 
 If the command returns one of the affected versions, disable the `rke2-ingress-nginx-admission` validating webhook configuration by performing the following steps:
 
-1. On one of your control plane nodes, create the file `/var/lib/rancher/rke2/server/manifests/rke2-ingress-nginx-config.yaml`.
+1. On one of your control plane nodes, use `kubectl` to confirm the existence of the `HelmChartConfig` resource named `rke2-ingress-nginx`:
+
+   ```sh
+   $ kubectl -n kube-system get helmchartconfig rke2-ingress-nginx
+   NAME                 AGE
+   rke2-ingress-nginx   14d1h
+   ```
+
+1. Use `kubectl -n kube-system edit helmchartconfig rke2-ingress-nginx` to add the following configurations to the resource:
```

Comment:
`kubectl patch` won't work here because the value of `.spec.valuesContent` is a multi-line string blob.